lib/csv/writer.rb

```diff
@@ -18,6 +18,7 @@ def initialize(output, options)
       if @options[:write_headers] and @headers
         self << @headers
       end
+      @converters = @options[:fields_converter] unless @options[:fields_converter].empty?
```

Comment:
This is a converter not converter"s".
`unless @options[:fields_converter].empty?` is needless.

Reply:
Hey @kou, thanks for the feedback!

I named it `converters` because this instance (FieldsConverter) can hold zero, one or many converters. Do you think it's better if I just name it `@converter` or should the variable be named as `fields_converter`? I think the second option explicit tells any other developer what instance they are handling.

Regarding the `.empty?` call, if I don't add it, we will call `@converters.convert`  regardless of any converter informed by the user. It is not a call to `Hash#empty?` but it is a call to `FieldsConverter#empty?`.

Reply:
`@fields_converter` is better because we use `@fields_converter` in `CSV::Parser`.

OK. If we can confirm the `empty?` check optimization has performance improvement by benchmark, we can accept it.

NOTE: We may accept more new converters after we instantiate writer object. `CSV#convert` is a method that adds a new converter on parsing after we instantiate parser object. We may add `CSV#write_convert` method in the future. We need to remove the `empty?` check optimization when we add the method.

Reply:
@kou finished the suggestions.

I have changed the variable name and removed the `.empty?` check. I also had to add the base converters so nil values are converted to empty string.

Reply:
OK. But we should use different names for them such as `write_empty_value` and `write_nil_value`. Because context is different on parsing and writing.
I'll do this after I merge this.